### PR TITLE
Move urls.SLUG_REGEXP to constants

### PR DESCRIPTION
**Pull request by mikek**
This fixes the problem with Django 1.9 system checks. In a few words:
importing from cms.urls triggers app resolver mechanism that fails
if there were no database tables for `contrib.sites` created yet.
It catches `DoesNotExist` but not `OperationalError`. Moving this
constant out to a different location seems to be a better fix.

### CODE REVIEW

File: cms/urls.py [hunk 1]
```diff
@@ -4,10 +4,9 @@
 
 from cms.apphook_pool import apphook_pool
 from cms.appresolver import get_app_patterns
+from cms.constants import SLUG_REGEXP
 from cms.views import details
 
-# This is a constant, really, but must live here due to import order
```
Comment (mikek) [inline cms/urls.py]: @yakky can you, please check if this is really the issue or move this constant to some other place instead?

**Comment by Chive** [comment]
:+1:

**Comment by yakky** [comment]
LGTM. A lot of circular import fixes since that comments.
Merged through #4995

**Comment by yakky** [comment]
Fixed by #4995

**Pull request closed without merging**